.meta/releases/0.7.0.toml

```diff
@@ -0,0 +1,365 @@
+  <li> `clickhouse` </li>
+  <li> `datadog_metrics` </li>
+  <li> `elasticsearch` </li>
+  <li> `gcp_pubsub` </li>
+  <li> `http` </li>
+  <li> `new_relic_logs` </li>
+  <li> `splunk_hec` </li>
+</ul>
+
+<p>
+In order to migrate all fields prefixed with `request_` must be placed within a
+`request` table with the prefix removed.
+
+The config field `retry_backoff_secs` must also be replaced with two new fields
+`retry_initial_backoff_secs` and `retry_max_duration_secs`.
+
+Fields prefixed with `batch_` must be placed within a `batch` table with the
+prefix removed. Instances of `batch_size` should be renamed `max_size` or
+`max_events` (refer to the relevant sink docs for the correct variant).
+
+Finally, the table `basic_auth` should renamed `auth` with a field `strategy`
+added:
+</p>
+
+<pre>
+ [sinks.my_sink_id]
+   type = "http"
+   inputs = ["my-source-id"]
+   uri = "https://10.22.212.22:9000/endpoint"
+
+-  batch_size = 1049000
++  [sinks.my_sink_id.batch]
++    max_size = 1049000
+
+-  [sinks.my_sink_id.basic_auth]
++  [sinks.my_sink_id.auth]
++    strategy = "basic"
+     user = "${USERNAME_ENV_VAR}"
+     password = "${PASSWORD_ENV_VAR}"
+
+-  request_in_flight_limit = 5
+-  request_retry_backoff_secs = 1
++  [sinks.my_sink_id.request]
++    in_flight_limit = 5
++    retry_initial_backoff_secs = 1
++    retry_max_duration_secs = 10
+</pre>
+"""
+
+[[releases."0.7.0".upgrade_guides]]
+id = "buffer-num-items-changed"
+commits = []
+title = "The buffer field `num_items` has been renamed to `max_events`"
+body = """\
+<p>
+The buffer field `num_items` has been renamed to `max_events` in order to clarify
+its purpose.
+
+Migration involves simply changing all occurrences of `buffer.num_items` to
+`buffer.max_events`:
+</p>
+
+<pre>
+ [sinks.my_sink_id]
+   type = "aws_s3"
+   inputs = ["my-source-id"]
+   bucket = "my-bucket"
+
+   [sinks.my_sink_id.buffer]
+     type = "memory"
+-    num_items = 500
++    max_events = 500
+</pre>
+"""
+
+[[releases."0.7.0".upgrade_guides]]
+id = "kafka-sink-comma-list"
+commits = []
+title = "The `kafka` sink now takes a comma separated list of bootstrap servers"
+body = """\
+<p>
+The `kafka` sink field `bootstrap_servers` has been changed from an array to a
+string, expecting a comma separated list of bootstrap servers similar to the
+`kafka` source:
+</p>
+
+<pre>
+ [sinks.my_sink_id]
+   type = "kafka"
+   inputs = ["my-source-id"]
+-  bootstrap_servers = ["10.14.22.123:9092", "10.14.23.332:9092"]
++  bootstrap_servers = "10.14.22.123:9092,10.14.23.332:9092"
+</pre>
+"""
+
+[[releases."0.7.0".upgrade_guides]]
+id = "journald-source-switch"
+commits = []
+title = "The `journald` source now pipes from journalctl"
+body = """\
+<p>
+The `journald` source originally used libsystemd in order to consume logs from
+Journald. Since this was causing issues for some distributions of Vector it has
+now been changed to pipe logs from journalctl.
+</p>
+"""
+
+[[releases."0.7.0".upgrade_guides]]
```

Comment:
I think we can remove this one. This is a _very_ new source that hasn't been advertised, no one is using it. Removing this will help reduce the noise around upgrading.